data/food/export_agb/export_builder.py

```diff
@@ -21,6 +20,7 @@
 BIOSPHERE = DBNAME + " biosphere"
 PROCESSES2EXPORT = "builder_processes_to_export.csv"
 INGREDIENTS_BASE = "ingredients_base.json"
+IMPACTS = "../../../public/data/impacts.json"  # TODO move the impact definition somewhere else and remove base impact
```

Comment:
FYI in https://github.com/MTES-MCT/ecobalyse/pull/298 we're getting rid of this file altogether ;)